src/nl/hannahsten/texifyidea/util/parser/Psi.kt

```diff
@@ -179,6 +180,15 @@ fun PsiElement.inComment() = inDirectEnvironmentContext(Environment.Context.COMM
     else -> this is LeafPsiElement && elementType == LatexTypes.COMMAND_TOKEN
 }
 
+/**
+ * Check if the element is in a url command.
+ */
+fun PsiElement?.inUrl(): Boolean {
```

Comment:
I think we should generalize at least to verbatim commands in general, not just urls (`\url` is also parsed as raw text). Though really we should ignore `&` if they are not at the same parsing level (ignore a table-in-table), but not sure if that's easy to do